samza-test/src/test/java/org/apache/samza/test/table/TestLocalTableWithSideInputs.java

```diff
@@ -28,10 +28,7 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 import org.apache.samza.application.StreamApplication;
-import org.apache.samza.config.Config;
-import org.apache.samza.config.JobConfig;
-import org.apache.samza.config.MapConfig;
-import org.apache.samza.config.StreamConfig;
+import org.apache.samza.config.*;
```

Comment:
Why import *?

Reply:
Intellij default importing reduced it to .* . 

Now fixed.